README.adoc

```diff
@@ -8,26 +8,9 @@ GitHub repo for SteeltoeOSS Docker images.
 
 == Building
 
-.Build all images (not yet implemented)
-----
-$ build.ps1 -All
-----
-
 .Build a specific image
 ----
-$ build.ps1 config-server
-----
-
-== Pushing to Docker Hub
-
-.Push a specific image
-----
-$ docker push steeltoeoss/config-server:1.2.3-linux
-----
-
-.Push a manifest
-----
-$ manifest-tool --username * --password * push from-spec config-server/manifest.yml
+./build.ps1 config-server
```

Comment:
For Steeltoe users unfamiliar with docker, it would help to explain how to start a container from the locally built image and expose its port, ie:
```
docker images
```
```
docker run --rm -ti -p 8888:8888 --name steeltoe-config steeltoeoss/config-server
```
Possibly add a link to https://github.com/SteeltoeOSS/Samples/blob/main/CommonTasks.md. And the content there should probably be updated as well, to not use steeltoe images anymore for postgres/mysql/mssql etc.

_Originally posted by @bart-vmware in https://github.com/SteeltoeOSS/Dockerfiles/pull/32#discussion_r1502411491_

---

As far as general instruction goes, I agree it could be better... I was planning to come back through (when we are 100% sure of which docker images we want to be shipping) during the connecting-the-dots efforts after either the milestone/preview or GA of v4 and make sure all of the metadata is in good shape both in the repo and on docker hub.

_Originally posted by @TimHess in https://github.com/SteeltoeOSS/Dockerfiles/pull/32#discussion_r1502755361_

---

As discussed, we should at least link to the Wiki where all the docker parameters are listed. How about adding the following:

```
== Running

You can list the created images using:
----
docker images
----

See the link:https://github.com/SteeltoeOSS/Samples/blob/main/CommonTasks.md/[docker instructions] on how to run the various docker images.
```